src/main/java/core/basesyntax/service/impl/CreateReportServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.CreateReportService;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.TransactionParseService;
+import java.util.Map;
+
+public class CreateReportServiceImpl implements CreateReportService {
+    private static final String TITLE = "fruit,quantity";
+    private final TransactionParseService transactionParseService;
+    private final FruitShopService fruitShopService;

```

Comment:
You do not use `TransactionParseService` and `FruitShopService` in this class.
Please, remove it